### PR TITLE
Recipes: ensure unique topic names

### LINKED ISSUE #1089: Recipes: ensure unique topic names
A small number of recipes overlap source topic names, so there may be clashes if a user runs 2 of those recipes serially.

Action: analyze all recipes and create unique names, or namespace the topics per recipe

### PR DESCRIPTION
**Pull request by ybyzek**
### Description

#1089 

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/resolve-topic-clash

### CODE REVIEW

File: _includes/tutorials/discount-promo/confluent/code/tutorial-steps/dev/source.json [hunk 1]
```diff
@@ -25,9 +25,9 @@
   "connection.user"          : "postgres",
   "connection.password"      : "<my-database-password>",
   "db.name"                  : "<db-name>",
-  "table.whitelist"          : "orders",
+  "table.whitelist"          : "order_stream",
```
Comment (rspurgeon) [inline _includes/tutorials/discount-promo/confluent/code/tutorial-steps/dev/source.json:28]: Nit: Raising the possibility of having a table with the name `stream` could be confusing. FYC name this something very generic like `order_data` or similar

Comment (bbejeck) [reply _includes/tutorials/discount-promo/confluent/code/tutorial-steps/dev/source.json:28]: +1

**Review by bbejeck — COMMENTED** [review]
Changes LGTM modulo comments from @rspurgeon

File: _includes/tutorials/discount-promo/confluent/code/tutorial-steps/dev/source.json (hunk 1 above)
Comment (ybyzek) [reply _includes/tutorials/discount-promo/confluent/code/tutorial-steps/dev/source.json]: Agree and fixed in latest commit

**Review by rspurgeon — APPROVED** [review]
lgtm

**Pull request merged**